Assets/Scripts/ProceduralGeneration.cs

```diff
@@ -31,6 +31,9 @@ public class ProceduralGeneration : MonoBehaviour
 
     // Indices match up to values of EntityType enum
     public float[] spawnProbability;
+    [Tooltip("Will there be a black hole to chase the player")]
+    public bool chaseLevel = true;
```

Comment:
Can you revert the changes in this file to avoid merge conflicts? I did the same exact thing in my branch (except I called it `disableBlackHole`, because it's also useful for debugging)

Reply:
Done